gen/src/Gen/AST/Data/Syntax.hs

```diff
@@ -374,7 +375,8 @@ requestD c m h (a, as) (b, bs) =
     (identifier a)
     $ Just
       [ assocD (identifier a) "AWSResponse" (typeId (identifier b)),
-        funD "request" (requestF c m h a as),
+        funArgsD "service" ["_"] (var (m ^. serviceConfig)),
+        funArgsD "request" ["srv"] (requestF c m h a as),
```

Comment:
We could try to point-free this in the common case, but then we need to think harder when plugins are involved. Worth it?

Reply:
Nobody seems to think it's worth it.